service/worker/scheduler/workflow.go

```diff
@@ -205,10 +206,11 @@ var (
 		IterationsBeforeContinueAsNew:     0,
 		SleepWhilePaused:                  true,
 		MaxBufferSize:                     1000,
+		BackfillsPerIteration:             10,
 		AllowZeroSleep:                    true,
 		ReuseTimer:                        true,
 		NextTimeCacheV2Size:               14,                   // see note below
-		Version:                           DontTrackOverlapping, // TODO: upgrade to InclusiveBackfillStartTime
+		Version:                           DontTrackOverlapping, // TODO: upgrade to IncrementalBackfill
```

Comment:
When will this no longer be a TODO?

Reply:
We have to make changes to workflow logic in two stages, write the logic first but not enabled, then release that, then enable it, then release that. Otherwise we can't roll back an upgrade without possibly breaking schedules that made an update during that time, which would add too much risk to releases.